ngraph/core/src/op/bucketize.cpp

```diff
@@ -8,7 +8,7 @@
 using namespace ngraph;
 using namespace std;
 
-constexpr NodeTypeInfo op::v3::Bucketize::type_info;
+NGRAPH_RTTI_DEFINITION(op::v3::Bucketize, "Bucketize", 3);
 
```

Comment:
@ilyachur should 4th parameter be provided and what should it be, if operator inherits from Op class, like in example below?
`class NGRAPH_API Bucketize : public Op`

Reply:
No we shouldn't. Op class doesn't have RTTI declaration